tests/stylus/StylintBearTest.py

```diff
@@ -0,0 +1,233 @@
+import os
+from queue import Queue
+
+from bears.stylus.StylintBear import StylintBear
+from coalib.results.Result import Result
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from coalib.settings.Section import Section
+from coalib.testing.BearTestHelper import generate_skip_decorator
+from coalib.testing.LocalBearTestHelper import LocalBearTestHelper
+from coala_utils.ContextManagers import prepare_file
+
+test_bad_missing_colon = """
+test = @block
+    margin 0
+"""
```

Comment:
hm just thinking about it: Loading testfiles is also something you could address. So you would enforce a standard for directory names and path structures, so people can load test files very easily.
Although this is okay, it would shrink the amount of test code someone has to read if you split up those into test files. I think we should keep this for now, we can talk about practices for test files later ;)